pkgs/development/compilers/openjdk/17.nix

```diff
@@ -93,7 +93,7 @@ let
 
     configureFlags = [
       "--with-boot-jdk=${openjdk-bootstrap.home}"
-      "--with-version-build=${version.build}"
+      #"--with-version-build=${version.build}"
```

Comment:
Can you just confirm that the reported version is correct without setting this?

Reply:
I’m still building :)

But FWIW the other `-ga` versions, including 22 before this PR, don’t pass this flag, and the Temurin build system omits it for `-ga`, so I believe this is correct.

Reply:
We could just use the `+7` version rather than `-ga` since it’s the same tag, I guess. I don’t really know what’s best here.

Reply:
> We could just use the +7 version rather than -ga since it’s the same tag, I guess. I don’t really know what’s best here.

I think I am more comfortable to switch to the non-ga versions, but I am fine for whatever you decide here.

Reply:
OK, done. Didn’t touch 8 or 23 since they were already like that. We should just be consistent with Temurin but it’ll have to wait until after refactoring. It’s too many builds for me to bother running again, but this should be consistent with the previous state.